extensions/wikia/CommunityPage/templates/topContributors.mustache

```diff
@@ -19,7 +19,7 @@
 				<a href="{{profilePage}}">{{{avatar}}}</a>
 			</div>
 			<div class="contributor-details">
-				{{count}}. <a href="{{profilePage}}">{{userName}}</a>
+				{{count}}. <a data-tracking="top-contributors" href="{{profilePage}}">{{userName}}</a>
```

Comment:
I think format like top-contributors-{modal/module}-user will be more clear which means we would need to provide different label depending whether topContributors template is used in module or modal.

What do you think about it?